pages/contribute/rule-feedback.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
```

Comment:
```suggestion
Feedback is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
```

I think the first paragraph on this page should cover all types of feedback, not only on rules.